app/middleware/get-service-and-gateway-account.middleware.js

```diff
@@ -34,16 +33,11 @@ async function getGatewayAccountByExternalId (gatewayAccountExternalId, correlat
   } catch (err) {
     const logContext = {}
     logContext['error'] = err.message
+    logContext['error_code'] = err.errorCode
     logContext['GATEWAY_ACCOUNT_EXTERNAL_ID'] = gatewayAccountExternalId
     logContext[keys.CORRELATION_ID] = correlationId
```

Comment:
Can probably skip all this logging for an unexpected error. The error handler will log enough and we can trace together exactly what went wrong by following through and looking at the connector logs

Reply:
Can ignore, it would make sense to log something here probably. Just need to make sure it's only logging at error level if it's a problem with the system that we need to address

Reply:
Updated logging. Logs at error level only if there is a problem and info level when not found.